packages/dcos-ui-service/buildinfo.json

```diff
@@ -0,0 +1,8 @@
+{
+  "single_source": {
+    "kind": "url_extract",
```

Comment:
`pkgpanda.build.BuildError: Validation error when fetching sources for package: Unsupported archive: dcos-ui-update-service`

it's not archive, no need to extract. Looks like there's `"kind": "url"` that we should use instead

@GeorgiSTodorov